### DISCUSSION TITLE
Allow users to type in the amount of comments to scan for a specific video

### DISCUSSION THREAD

**Discussion by NotDreppy** [ThioJoe/YT-Spammer-Purge#376] [Ideas]
Some videos have a lot of spam comments, especially for million-sub YouTubers, and sometimes the spam comments for 1 single video are over 10,000. I hope we can add the feature to scan ONE PART of all the comments in a video, then the latest comments won't be spam or scam and we won't have to use up our whole youtube quota without having any comments staying!

**Comment by ThioJoe** [comment]
Yea i'm working on that
